UserInterface/Views/SettingsView.xaml

```diff
@@ -0,0 +1,89 @@
+﻿<UserControl x:Class="UserInterface.Views.SettingsView"
+             xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008" 
+             xmlns:local="clr-namespace:UserInterface.Views"
+             xmlns:wpfx="http://schemas.xceed.com/wpf/xaml/toolkit"
+             xmlns:viewmodels="clr-namespace:UserInterface.ViewModels" 
+             d:DataContext="{d:DesignInstance Type=viewmodels:SettingsViewModel}"
+             mc:Ignorable="d"
+             >
+
+
+    <Border Style="{StaticResource ContentContainer_XXL}">
+        <Grid>
+            <Grid.RowDefinitions>
+                <RowDefinition Height="65"/>
+                <RowDefinition Height="15"/>
+                <RowDefinition Height="90"/>
+                <RowDefinition Height="5"/>
+                <RowDefinition Height="*"/>
+                <RowDefinition Height="90"/>
+            </Grid.RowDefinitions>
+            <Grid.ColumnDefinitions>
+                <ColumnDefinition Width="*"/>
+                <ColumnDefinition Width="*"/>
+                <ColumnDefinition Width="*"/>
+            </Grid.ColumnDefinitions>
+
+            <TextBlock 
+                Style="{StaticResource Header}"
+                HorizontalAlignment="Center"  
+                Text="Applicatie instellingen" 
+                Grid.Row="0"
+                Grid.ColumnSpan="3"/>
+            <Rectangle 
+                Grid.Row="1" 
+                Grid.Column="0" 
+                Grid.ColumnSpan="3" 
+                Fill="{DynamicResource SecondaryColor}"/>
+
+            <TextBlock
+                Style="{StaticResource Text}"
+                Text="Thema instelling"
+                Grid.Column="0"
+                VerticalAlignment="Center"
+                Grid.Row="2"
+                Padding="25 10"/>
+            <Rectangle
+                Grid.Column="1"
+                Grid.Row="2"
+                RadiusX="30"
+                RadiusY="30"
+                Height="60"
+                Fill="{DynamicResource SecondaryBackgroundColor}"/>
+            <Rectangle
+                Grid.Column="1"
+                Grid.Row="2"
+                Height="25"
+                Width="320"
+                Fill="{Binding SavedColor}"/>
+            <wpfx:ColorPicker 
```

Comment:
misschien is het nog mooi om de groote van de color picker aan te passen:
 <wpfx:ColorPicker 
   Name="ColorPicker" 
   Height="70" 
   Width="300"
   Grid.Row="2"
   HorizontalAlignment="Right"
   Grid.Column="2"
   FontSize="28"
   DisplayColorAndName="True" 
   Margin="0 0 50 0"   
   SelectedColor="{Binding SelectedColor}"
   ShowTabHeaders="True">
 </wpfx:ColorPicker>
 ?

Reply:
In dat stukje is het ook mogelijk om de standard weg te houden want het is erg veel en niet perse nodig.